queen-attack/example.js

```diff
@@ -0,0 +1,47 @@
+const W = 8,
+  H = 8,
+  STARTING = { black : [7,3], white: [0,3] };
+
+const QueenAttack = ( params = STARTING )  => {
```

Comment:
There is a very good opportunity here to use *default parameters* and *deconstruction*:

```
const QueenAttack = ({ black = [7, 3], white = [0, 3] }) => {
// ...
```

Just in case you want to create another PR :stuck_out_tongue_winking_eye: